massa-proto/proto/massa/api/v1/api.proto

```diff
@@ -414,6 +420,19 @@ message SelectorDraws {
   repeated IndexedSlot next_endorsement_draws = 3;
 }
 
+message GetMipStatusRequest {
+  // Request id
+  string id = 1;
+}
+
+// GetMipStatusResponse holds response from GetMipStatus
+message GetMipStatusResponse {
+  // Request id
+  string id = 1;
+  // MipInfo - status id entry
+  repeated MipStatusEntry status = 2;
```

Comment:
IMHO should be renamed to `entry` to be coherent

Reply:
done